waspc/src/Wasp/Project/ExternalConfig/PackageJson.hs

```diff
@@ -18,11 +18,11 @@ import Wasp.Project.Common
   )
 import qualified Wasp.Util.IO as IOUtil
 
-analyzePackageJsonFile :: Path' Abs (Dir WaspProjectDir) -> IO (Either [String] P.PackageJson)
-analyzePackageJsonFile waspProjectDir = runExceptT $ do
+analyzePackageJsonFile :: Bool -> Path' Abs (Dir WaspProjectDir) -> IO (Either [String] P.PackageJson)
```

Comment:
We should probably rework this signature also, with similar approach as to how I suggested in other comments.